ttmar_starter.html

```diff
@@ -61,6 +61,42 @@
 
   <script>
 
+
+  document.getElementById("first_paragraph").addEventListener("click",function() {
+    document.getElementsByTagName('body')[0].style.backgroundColor = "gray"
+  })
+
+  document.getElementById("second_paragraph").addEventListener("click",function() {
+    event.preventDefault();
+    document.getElementsByClassName("second_paragraph")[0].style.display = "block"
+  })
+
+  document.getElementById("first_chorus").addEventListener("click",function() {
+    event.preventDefault();
+    for (var i=0; i<document.getElementsByClassName("first_chorus").length;i++){
+    document.getElementsByClassName("first_chorus")[i].style.display = "block"};
+  })
+
+  document.getElementById("third_paragraph").addEventListener("click",function() {
+    event.preventDefault();
+    for (var i=0; i<document.getElementsByClassName("third_paragraph").length;i++){
+    document.getElementsByClassName("third_paragraph")[i].style.display = "block"};
+  })
+
+  document.getElementById("fourth_paragraph").addEventListener("click",function() {
+    event.preventDefault();
+    for (var i=0; i<document.getElementsByClassName("fourth_paragraph").length;i++){
+    document.getElementsByClassName("fourth_paragraph")[i].style.display = "block"};
+  })
+
+    var counter=0;
+
+  document.getElementById("second_chorus").addEventListener("click",function() {
+    event.preventDefault();
+    counter = counter+1;
+    document.getElementsByClassName("second_chorus")[counter-1].style.display = "block";
```

Comment:
this is really a small comment, but if you swap these lines and increment counter after you change the style, you won't have to do `counter-1` and can just put counter.